packages/docs-app/src/components/navbarActions.tsx

```diff
@@ -63,9 +63,20 @@ export class NavbarActions extends React.PureComponent<INavbarActionsProps, {}>
      * Also include a link to the GitHub release notes.
      */
     private renderReleasesMenu() {
-        const menuItems = this.props.releases.map((version: IPackageInfo, index: number) => (
+        const { releases } = this.props;
+        const renderItem = (version: IPackageInfo, index: number) => (
             <MenuItem href={version.url} key={index} label={version.version} target="_blank" text={version.name} />
-        ));
+        );
+        const COMPONENT_PACKAGES = [
+            "@blueprintjs/core",
+            "@blueprintjs/datetime",
+            "@blueprintjs/table",
+            "@blueprintjs/labs",
+        ];
+        const libs = releases.filter(({ name }: IPackageInfo) => COMPONENT_PACKAGES.indexOf(name) >= 0).map(renderItem);
```

Comment:
shouldn't need `: PackageInfo` param type

Reply:
pretty sure you almost always do when destructuring in lambdas?

Reply:
nvm you're right